feature-pack/embedded/src/main/resources/modules/org/infinispan/core/slot/module.xml

```diff
@@ -12,7 +12,7 @@
         <module name="org.apache.xerces" services="import"/>
         <module name="org.infinispan.commons" slot="@infinispan.module.slot@" />
         <module name="com.github.ben-manes.caffeine" slot="@infinispan.module.slot@"/>
-        <!--Export rxjava and reactive streams for store implementations-->
+        <!--Export rxjava and store streams for store implementations-->
```

Comment:
I'm pretty sure it should be `reactive streams` here :)

Reply:
thnaks to intellij
I will redo things here anyway